api/api.yaml

```diff
@@ -215,16 +215,27 @@ paths:
           $ref: '../result/request.yaml#/components/responses/InternalServerError'
 
   /results/state:
+    description: >-
+      State API
```

Comment:
State or States?

Reply:
reason why I ask is if we keep this singular, while Metrics is plural.  Shouldn't we be consistent?
@PrasenjitAdhikary @ankur-sheth ?